osgp/shared/osgp-ws-publiclighting/src/main/resources/schemas/pl-adhocmanagement.xsd

```diff
@@ -102,7 +102,10 @@
         <xsd:element name="Result" type="common:OsgpResultType"
           minOccurs="1" />
         <xsd:element name="Description" type="xsd:string" />
-        <xsd:element name="DeviceStatus" type="tns:DeviceStatus" />
+        <xsd:choice>
+            <xsd:element name="DeviceStatus" type="tns:DeviceStatus" />
+            <xsd:element name="LightSensorStatus" type="tns:LightSensorStatus" />
```

Comment:
Indented by 4 instead of 2 spaces.

The other thing I was wondering about is, what the difference would be if this were a `Status` (new type) instead of the choice. Then `DeviceStatus` and `LightSensorStatus` could extend `Status`. Do you know if this would result in nicer code or not?